scrapeArtists/artistsFromRecordings_example.json

```diff
@@ -0,0 +1 @@
+{"table":[{"artistId":"e8c91353-e251-456b-80ad-d10c2cd00be3","aboutartist":{"gender":null,"isnis":[],"id":"e8c91353-e251-456b-80ad-d10c2cd00be3","life-span":{"ended":false,"end":null,"begin":null},"name":"University of Arkansas Chorus","begin_area":null,"type":"Choir","sort-name":"University of Arkansas Chorus","country":null,"area":{"id":"8788d6c2-c779-4be5-ad47-cf0a95e0f2a0","disambiguation":"","iso-3166-2-codes":["US-AR"],"name":"Arkansas","sort-name":"Arkansas"},"end_area":null,"gender-id":null,"disambiguation":"","type-id":"6124967d-7e3a-3eba-b642-c9a2ffb44d94","ipis":[]}},{"artistId":"69efc63a-6bf3-4763-8a30-cf7fd66995f6","aboutartist":{"type-id":"b6e035f4-3ce9-331c-97df-83397230b0df","ipis":[],"area":{"id":"489ce91b-6658-3307-9877-795b68554c98","iso-3166-1-codes":["US"],"disambiguation":"","name":"United States","sort-name":"United States"},"end_area":null,"gender-id":"36d3d30a-839d-3eda-8cb3-29be4384e4a9","disambiguation":"","life-span":{"begin":null,"end":null,"ended":false},"name":"Jack Groh","begin_area":null,"sort-name":"Groh, Jack","type":"Person","country":"US","gender":"Male","isnis":[],"id":"69efc63a-6bf3-4763-8a30-cf7fd66995f6"}}]}
```

Comment:
Please don't commit these json files. It only blows up your Git History and makes it slow.